docs/articles/highway-sdk/objects.md

```diff
@@ -1,57 +0,0 @@
----
```

Comment:
Do we know why these files were removed?

Reply:
they where not really applicable at this time and the information is now within each module